PKHeX.WinForms/Subforms/PKM Editors/MemoryAmie.cs

```diff
@@ -435,10 +435,10 @@ private void changeCountryText(object sender, EventArgs e)
 
         private void update255_MTB(object sender, EventArgs e)
         {
-            MaskedTextBox mtb = sender as MaskedTextBox;
+            MaskedTextBox mtb = (MaskedTextBox) sender;
```

Comment:
Hm, I'll dig deeper into our usage, but regardless of the overall choice - there are some cases where we should be able to migrate one to the other safely.